app/pages/api/auth/[...nextauth].ts

```diff
@@ -189,7 +190,14 @@ export const authOptions: NextAuthOptions = {
     PatreonProvider({
       clientId: process.env.PATREON_ID,
       clientSecret: process.env.PATREON_SECRET,
-    })
+    }),
+    AsgardeoProvider({
+      domain: process.env.ASGARDEO_DOMAIN,
+      tenantId: process.env.ASGARDEO_TENANT,
```

Comment:
Is there any reason why we need both domain and tenant?

Reply:
also shouldn't tenantId --> organizationName?

Reply:
Hi @mefarazath, Yes can proceed with organization name and doesnt require domain.
Sry for the delay :)